testing/scenario_app/android/app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,23 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".ExternalTextureFlutterActivity"
+            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
+            android:hardwareAccelerated="true"
+            android:launchMode="singleTop"
+            android:windowSoftInputMode="adjustResize"
+            android:theme="@style/FullScreenScreenshot">
+            <intent-filter>
+                <action android:name="com.google.intent.action.TEST_LOOP" />
```

Comment:
this seems correct. I wonder if the issue is that only one activity can have this action. What happens if you remove the content of the outer `<intent-filter>`?